admin-dev/themes/new-theme/js/components/modal/confirm-modal.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+
+/* eslint max-classes-per-file: ["error", 2] */
+
+import {
+  ModalContainerType, ModalContainer, ModalType, ModalParams, Modal,
+} from '@components/modal/modal';
```

Comment:
```suggestion
} from '@components/modal';
```

And renaming the modal.ts to index.ts, wdyt?

Reply:
Actually we can't, because there is already a `@components/modal.ts` file that export all available modals
The idea is that you can do things like this:

`import {ConfirmModal} from '@components/modal';`
`import {IframeModal, Modal} from '@components/modal';`

and also `import ConfirmModal from '@components/modal';` as `ConfirmModal` is the default module in this file to keep backward compatibility with previous imports